lms/templates/student_account/login_and_register.html

```diff
@@ -1,6 +1,7 @@
 <%!
```

Comment:
I think this quality check is not included in `run_quality` for some reason. You have to explicitly run `paver run_safecommit_report`. See [safe template linter](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#safe-template-linter) for more details.